docs/js-core-1/week-3/learning-objective.md

```diff
@@ -0,0 +1,22 @@
+## PD (team-working and communication)
```

Comment:
Again, I really recommend splitting this up and making it more relevant to the content.

It will also help improve the learning objectives, if you more clearly assign them to exercises.

https://teachtogether.tech/en/index.html#s:process-objectives